src/background.js

```diff
@@ -21,3 +19,26 @@ chrome.tabs.onRemoved.addListener((tabId) => {
     delete activeTabUrlMemo[tabId]
   }
 })
+
+chrome.runtime.onInstalled.addListener(() => {
```

Comment:
Cool!!
btw, maybe we can add the contextMenu option only on `non-GitHub domain` pages?

Reply:
not sure if it is possible (or necessary